easybuild/easyblocks/s/scotch.py

```diff
@@ -108,7 +117,7 @@ def build_step(self):
         if self.toolchain.options['i8']:
             cflags += " -DINTSIZE64"
 
-        if not self.toolchain.mpi_family() in [toolchain.INTELMPI, toolchain.QLOGICMPI]:  #@UndefinedVariable
+        if self.cfg['threadedmpi']: 
```

Comment:
Ok, but I'm pretty sure this wasn't working for people, to my eyes it's mostly there for OpenFOAM and threaded scotch does not work with OpenFOAM.
I'll set the default to true for OpenMPI but personally I think this is probably not doing what people actually expect.